src/geocat/comp/interp_hybrid_to_pressure.py

```diff
@@ -0,0 +1,111 @@
+import metpy.interpolate
+import numpy as np
+import xarray as xr
+
+__pres_lev_mandatory__ = np.array([
+    1000, 925, 850, 700, 500, 400, 300, 250, 200, 150, 100, 70, 50, 30, 20, 10,
+    7, 5, 3, 2, 1
+])  # Mandatory pressure levels (mb)
+__pres_lev_mandatory__ = __pres_lev_mandatory__ * 100.0  # Convert mb to Pa
+
+
+def _pressure_from_hybrid(psfc, hya, hyb, p0=100000.):
+    """
+    Calculate pressure at the hybrid levels
+    """
+    # This will be in Pa
+    return hya * p0 + hyb * psfc
+
+
+def interp_hybrid_to_pressure(data,
+                              ps,
+                              hyam,
+                              hybm,
+                              p0=100000.,
+                              new_levels=__pres_lev_mandatory__,
+                              lev_dim='lev',
+                              method='linear'):
+    """
+    Interpolate data from hybrid-sigma levels to isobaric levels.
+
+    ACKNOWLEDGEMENT: We'd like to thank to Brian Medeiros (https://github.com/brianpm), Matthew Long
+    (https://github.com/matt-long), and Deepak Cherian (https://github.com/dcherian) at NCAR for their
+    great contributions since the code implemented here is mostly based on their work.
+
+    Parameters
+    ----------
+    data : `xarray.DataArray`:
```

Comment:
Per [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html), the back-ticks aren't necessary. 

For instance: 

```suggestion
    data : xarray.DataArray
```

Reply:
Done, thanks for suggesting this. (Will also address this in other scripts soon)